src/common/transformations/src/transformations/fix_rt_info.cpp

```diff
@@ -23,17 +22,12 @@ bool ov::pass::FixRtInfo::run_on_model(const std::shared_ptr<ngraph::Function>&
                 run_on_model(sub_graph);
             }
         }
-        OPENVINO_SUPPRESS_DEPRECATED_START
+        
         auto& rt_info = node->get_rt_info();
         {
             auto it_info = rt_info.find("PrimitivesPriority");
             if (it_info != rt_info.end()) {
-                if (it_info->second.is<std::shared_ptr<ngraph::VariantWrapper<std::string>>>()) {
-                    rt_info.emplace(
```

Comment:
Is there any problem if delete this branch?

Reply:
> Is there any problem if delete this branch?

Form code perspective, this branch operate with "VariantWrapper" seems like can be removed directly. @ilyachur  how do u think?

Reply:
@itikhono Could you comment?